main.js

```diff
@@ -35,6 +35,13 @@ var getMobileInfo = new Promise(function(resolve, reject) {
 
 var loadingPromises = [initFS, getMobileInfo];
 
+loadingPromises.push(load("java/classes.jar", "arraybuffer").then(function(data) {
+  CLASSES.addPath("java/classes.jar", data);
+  CLASSES.initializeBuiltinClasses();
+  // Load the Isolate class, that is needed to start the VM (see jvm.ts)
+  CLASSES.getClass("com/sun/cldc/isolate/Isolate");
```

Comment:
*initializeBuiltinClasses* both loads and links classes, but *getClass* only loads them. Can we link *com/sun/cldc/isolate/Isolate* here?